spongehome.go

```diff
@@ -76,8 +77,12 @@ func clearFastly() {
 
 func main() {
 	// Initialise Macaron
-	m := macaron.Classic()
```

Comment:
According to [the macaron docs](https://go-macaron.com/docs/middlewares/gzip), you can't use `macaron.Classic` if you want to gzip static files, since `macaron.Classic` automatically installs a static handler.